package.json

```diff
@@ -0,0 +1,81 @@
+{
+  "name": "egeria-ui",
+  "version": "0.0.1",
+  "description": "Egeria UI application",
+  "author": "Egeria contributors",
+  "license": "Apache-2.0",
+  "repository": "odpi/egeria-ui",
+  "vaadin": {
+    "disableUsageStatistics": true
+  },
+  "scripts": {
+    "serve": "hs -c0 ./build/dev/ -P http://localhost:9000",
+    "start": "polymer serve",
+    "build": "polymer build",
+    "lint": "polymer lint",
+    "test": "polymer test",
+    "test:integration": "polymer build # test that psk builds without error with the CLI"
+  },
+  "dependencies": {
+    "@polymer/app-layout": "^3.0.1",
+    "@polymer/app-localize-behavior": "^3.0.1",
+    "@polymer/app-route": "^3.0.1",
+    "@polymer/app-storage": "^3.0.2",
+    "@polymer/iron-a11y-keys": "^3.0.1",
+    "@polymer/iron-ajax": "^3.0.1",
+    "@polymer/iron-fit-behavior": "^3.0.2",
+    "@polymer/iron-flex-layout": "^3.0.1",
+    "@polymer/iron-form": "^3.0.1",
+    "@polymer/iron-icons": "^3.0.1",
+    "@polymer/iron-iconset-svg": "^3.0.1",
+    "@polymer/iron-label": "^3.0.1",
+    "@polymer/iron-localstorage": "^3.0.1",
+    "@polymer/iron-location": "^3.0.2",
+    "@polymer/iron-media-query": "^3.0.1",
+    "@polymer/iron-overlay-behavior": "^3.0.2",
+    "@polymer/iron-pages": "^3.0.1",
+    "@polymer/iron-selector": "^3.0.1",
+    "@polymer/neon-animation": "^3.0.1",
+    "@polymer/paper-button": "^3.0.1",
+    "@polymer/paper-checkbox": "^3.1.0",
+    "@polymer/paper-dialog": "^3.0.1",
+    "@polymer/paper-dialog-behavior": "^3.0.1",
+    "@polymer/paper-drawer-panel": "^3.0.2",
+    "@polymer/paper-dropdown-menu": "^3.0.1",
+    "@polymer/paper-icon-button": "^3.0.1",
+    "@polymer/paper-input": "^3.2.1",
+    "@polymer/paper-item": "^3.0.1",
+    "@polymer/paper-listbox": "^3.0.1",
+    "@polymer/paper-material": "^3.0.1",
+    "@polymer/paper-menu-button": "^3.0.1",
+    "@polymer/paper-progress": "^3.0.1",
+    "@polymer/paper-radio-button": "^3.0.1",
+    "@polymer/paper-radio-group": "^3.0.1",
+    "@polymer/paper-spinner": "^3.0.1",
+    "@polymer/paper-styles": "^3.0.1",
+    "@polymer/paper-toast": "^3.0.1",
+    "@polymer/paper-toggle-button": "^3.0.1",
+    "@polymer/polymer": "^3.4.1",
+    "@vaadin/vaadin": "^17.0.6",
+    "@vaadin/vaadin-button": "^2.2.0",
+    "@vaadin/vaadin-core": "^15.0.6",
+    "@vaadin/vaadin-date-picker": "^4.2.0",
+    "@vaadin/vaadin-dropdown-menu": "^1.2.0",
+    "@vaadin/vaadin-grid": "^5.6.6",
+    "@vaadin/vaadin-icons": "^4.3.1",
+    "@vaadin/vaadin-item": "^2.1.1",
+    "@vaadin/vaadin-list-box": "^1.2.0",
+    "@vaadin/vaadin-radio-button": "^1.2.3",
+    "@vaadin/vaadin-select": "^2.1.6",
+    "@vaadin/vaadin-split-layout": "^4.1.1",
+    "@vaadin/vaadin-tabs": "^3.0.5",
+    "@vaadin/vaadin-time-picker": "^2.2.0",
+    "@webcomponents/webcomponentsjs": "^2.1.3",
+    "multiselect-combo-box": "^2.4.2",
+    "visjs-network": "^4.24.11",
+    "web-animations-js": "^2.3.1"
+  },
+  "engines": {
+    "node": ">=8.0"
```

Comment:
this is way back.. I wonder if we should just increase to 12.x for safety?

Reply:
> this is way back.. I wonder if we should just increase to 12.x for safety?

in a future PR, i would want some unit tests first, cause we are walking on moving ground here..

Reply:
That's the reason I mentioned it -- currently the maven pom is building the UI in the official build with 12.18.2, which is why I thought restricting the spec here to at least 12 kept you closer to the version we build with all the time ;-)